latex/style.tt

```diff
@@ -1,5 +1,11 @@
 \usepackage{fontspec}
-\setmainfont{Helvetica Neue}
+\setmainfont{DroidSans}[
+  Extension   = .ttf,
+  UprightFont = *,
+  BoldFont    = *-Bold ]
+\setmonofont{DroidSansMono}[
```

Comment:
Is designating the mono font a forward-looking change for something else you want to add? The app has no mechanism for varying the typeface in a label.

Reply:
Ahh, yes, it is. Good deduction! The code 128 barcoding I have working locally (and will push next week once I sort out some janky bits) uses `\tt` for text underneath the barcode. I can rework this PR and move the monospace font definition to the barcode bits if you'd like.